Documentation/policy/language.rst

```diff
@@ -147,15 +147,16 @@ Services based
 --------------
 
 Services running in your cluster can be whitelisted in Egress rules. Currently
-headless Kubernetes services defined by their name and namespace or label selector
-are supported. More documentation on `HeadlessServices`.
-Future versions of Cilium will support specifying non Kubernetes services and services
-which are backed by pods.
+Kubernetes `HeadlessServices` are supported when defined by their name and
+namespace or label selector. Future versions of Cilium will support specifying
+non Kubernetes services and Kubernetes services which are backed by pods.
 
 This example shows how to allow all endpoints with the label ``id=app2``
 to talk to all endpoints of kubernetes service ``myservice`` in kubernetes
-namespace ``default``. Note that ``myservice`` needs to be a headless service
-for this policy to take effect.
+namespace ``default``.
+
+.. note::
+        ``myservice`` must be a headless service for this policy to take effect.
```

Comment:
This is no longer true, we support all "selector less" services now. Can you update it?
https://kubernetes.io/docs/concepts/services-networking/service/#services-without-selectors